alarm/utils.py

```diff
@@ -30,6 +37,26 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
                     f"threshold: {threshold}; "
                     f"threshold broken: {threshold_broken}")
         if threshold_broken:
+            ThresholdBrake.objects.create(threshold=threshold)
             return True
 
     return False
+
+
+def make_call(trade_pair):
+    threshold_prices = Threshold.objects.filter(trade_pair=trade_pair).values_list('price', flat=True)
+
+    formatted_prices_for_user_message = ', '.join([f'{price}$' for price in threshold_prices])
```

Comment:
Ok, naming is ok, I would still name it `threshold_prices_str`. It is shorter and still verbose.

Reply:
++